app/forms/spacecraft_registration_identifier_form.rb

```diff
@@ -0,0 +1,18 @@
+class SpacecraftRegistrationIdentifierForm
+  include ActiveModel::Model
+
+  attr_accessor :sri
+
+  def initialize(sri:)
+    @sri = sri
```

Comment:
What are your thoughts on using the full `spacecraft_registration_identifier` here? I'm a fan of avoiding acronyms where possible, but if it's common terminology throughout the service, maybe it's not a problem?

Reply:
good point, I also hate acronyms -- a huge source of obscurity...

Reviewing our See [Standard Terminology (ADR 12)](https://github.com/dxw/dfsseta-apply-for-landing-ruby/blob/main/doc/architecture/decisions/0012-standard-terminology.md?plain=1#L38-L39) I see that "Registration ID" is the right shortening, so I'm going to apply that...